HFP-3625 Change skipping behavior

otacke opened the pull request:
When merged in, will allow to control the skipping behavior more granular, [see acceptance criteria of HFP-3625](https://h5ptechnology.atlassian.net/browse/HFP-3625).

larsmagu commented:
@otacke Main functionality seems to work, however I've encountered what seems to be some artifacts. Please correct me if I'm mistaken. 

- The "go to time" button doesn't seem to work for skipping behavior "forward" and "forward and backwards". The user should be able to navigate forward in the video using this button for both behaviors. 
- After setting the behavior to "forward" or "forward and backwards" the user is not able to navigate forwards in the editor and add interactions later on. When changing back to "none" the user is still not able to navigate in editor (add interactions).

otacke commented:
@larsmagu You're probably right. I'll fix this as soon as I am back home.

otacke commented:
@larsmagu I fixed the issue with goToQuestion, but I cannot reproduce your trouble with the editor. I can navigate just fine and change things. Nevertheless, I added a guard to the function that checks whether the content type is in editing mode and then always allows skipping. Please let me know whether that fixes your issue or please give me more information in order to reproduce the behavior.

larsmagu commented:
Navigating the playbar seems to work in editor now, as well as the "go to" button. I'll do some additional testing and go over the code once more. Will let you know if I see any other issues. Also, I left a comment in the code regarding one of the parameters.

otacke commented:
@larsmagu Where's that comment?

otacke commented:
@larsmagu Sure does. Thanks for catching that.
